doc/mgr/orchestrator.rst

```diff
@@ -417,6 +418,10 @@ Where the properties of a service specification are the following:
 * ``service_id`` is the name of the service. Omit the service time
 * ``placement`` is a :ref:`orchestrator-cli-placement-spec`
 * ``spec``: additional specifications for a specific service.
+* ``unamanged``: If set to ``true``, the orchestrator will not deploy or
+   remove any daemons and will stop managing this service. If set to ``true``,
+   all other properties are ignored, including ``placement``. This is useful, if
+   this service should not be managed temporarily.
```

Comment:
```suggestion
* ``unmanaged``: If set to ``true``, the orchestrator will not deploy nor
   remove any daemon associated with this service. Likewise, all properties,
   including ``placement``, will be ignored. This is useful, if
   this service should not be managed temporarily.
```

Reply:
Instead of `Likewise, all properties, including placement, will be ignored.` - please use this variation `Placement and all other properties will be ignored.`

Reply:
fixed